### DISCUSSION TITLE
R9MM via r9mm_no_btn_elrs_bl.frk

### DISCUSSION THREAD

**Discussion by kt4rj** [ExpressLRS/ExpressLRS#1121] [Q&A]
Is there a different process for getting this to work compared to the normal process with bind button? I keep getting this error message.
![elrs no button fault](https://user-images.githubusercontent.com/29859039/142684024-0dc51861-f18b-4eec-aecc-99800b91451f.JPG)

**Comment by AlessandroAU** [comment]
Try clearing the configurator files (support -> clear platformio dependencies) and going again, it shouldn't make a difference.

**Reply by kt4rj** [reply]
Thank you Alessandro, I'll give that a try...

**Comment by AlessandroAU** [comment]
Also don't forget to update the configurator itself if you haven't
